tests/test_report_reader.cpp

```diff
@@ -4,6 +4,13 @@
 
 using namespace bbp::sonata;
 
+void testTimes(const std::vector<double>& vec, double start, double step, int size) {
+    REQUIRE(size == vec.size());
+    for (int i = 0; i < vec.size(); ++i) {
+        REQUIRE(std::abs(vec[i] - (start + step * i)) < std::numeric_limits<double>::epsilon());
```

Comment:
so there is no accumulation of error for this one ?

Reply:
it's read directly from file

Reply:
ok I see